website/events/templates/events/admin/registrations_table.html

```diff
@@ -51,15 +52,15 @@
                             data-none="no_payment"
                             {% if registration.payment %}class="paid"{% endif %}>
                         <option value="no_payment"
-                                {% if not registration.payment %}selected{% endif %}>{% trans 'None' %}</option>
+                                {% if not registration.payment %}selected{% endif %}>{% trans 'None' %} {% if registration.special_price %}(€ {{ registration.special_price }}) {% endif %}</option>
```

Comment:
Instead of repeating this price, can we put it in the same td but not in the select? This code is so ugly already :S

Reply:
I tried exactly that first, but it looked very ugly visually